draft-ietf-quic-ack-frequency.md

```diff
@@ -201,10 +201,9 @@ Sequence Number:
 ACK-eliciting threshold:
 
 : A variable-length integer representing the maximum number of ack-eliciting
-  packets after which the receiver sends an acknowledgement. A value of 1 will
-  result in an acknowledgement being sent for every ack-eliciting packet
-  received. A value of 0 is invalid. Receipt of an invalid value MUST be
-  treated as a connection error of type FRAME_ENCODING_ERROR.
+  packets the recipient of this frame can receive before sending an immediate
+  acknowledgment. A value of 0 will result in an immediate acknowledgement
+  whenever an ack-eliciting packet received.
```

Comment:
Is there a way to phrase this such that the off-by-one-ness is intuitive?
Something along the lines of "the number of packets after the most-recently-acked one to skip before sending another ack".

So saying "skip zero packets" means ack every time, saying "skip one packet after the most recently acked one" means ack every-other. 

Still trying to come up with a nice way to word that, but it seems like it could reduce off-by-one confusion.

Reply:
I'm definitely open to other wordings.  I think the example of 0 helps for me, but suggestions are welcome.

Reply:
The phrasing I used was "the number of packets you can receive without sending an immediate acknowledgment"

Reply:
The text is slightly but enough different to the suggestion you made, @martinthomson. I think we should replace "before" with "without".